db/schema.rb

```diff
@@ -67,6 +60,13 @@
     t.index ["user_id"], name: "index_sources_users_on_user_id"
   end
 
+  create_table "user_categories", force: :cascade do |t|
```

Comment:
Hmm, mas esse não vai gerar a tabela errada? Se não me engada, a parte importante até onde eu sei é conseguir gerar o `create_join_table` ali.

Ou isso é só para conseguir forçar a apagada dele na outra migration depois?